Latex/Insight.bib

```diff
@@ -3807,7 +3807,7 @@ @Article{Thirion1998
 
 @Book{Thom1977,
   Author         = "R. Thom",
-  Title          = "Stabilite Structurelle et Mofphogenese. Essai d'une
+  Title          = "Stabilite Structurelle et Morphogenese. Essai d'une
```

Comment:
Stabilit{\\'e}

Reply:
And on next lines:
th{\\'e}orie g{\\'e}n{\\'e}rale des mod{\\`e}les

Deuxi{\\`e}me

Reply:
Deuxi{\\`e}me : you missed the "e" within the curly brackets in be0b8e3.

Reply:
Oops, must have been a long day ...